contributing_to_docs/doc_guidelines.adoc

```diff
@@ -828,6 +828,37 @@ Embedding external files is restricted for files that change frequently, like te
 You must get approval from the Engineering, QE, and Docs teams before embedding an external file.
 ====
 
+== Embedding a local YAML file
+
+You can embed local YAML files in AsciiDoc modules.
+This is useful when you want to include a complete YAML custom resource in the docs.
+The YAML file that you include must be a local file maintained in the link:https://github.com/openshift/openshift-docs[openshift-docs] GitHub repository.
+Use the `include` directive to target the local file.
+
+To use a local YAML file, add it to the `/snippets` folder, and include it in your module. For example:
+
+[source,yaml]
+----
+\include::snippets/install-config.yaml[]
+----
+
+You can use callouts in the YAML file.
+Comment out the callout so that the YAML is valid. For example:
```

Comment:
I feel that this comment rules out line 845. Maybe a list of limitations should be listed before one considers including a YAML file in their module. From the OCP team meeting, a few limitations were mentioned. Maybe for simpler YAML examples, include a YAML file in a module, but more complex YAML examples (that include ifdef statements and possible callouts) should be directly placed in a code block in the module. WDYT?

Reply:
You comment out the codeblock in the YAML, so that the YAML remains valid. When it gets pulled into the AsciiDoc build, the build resolves the callout and applies the callout markup.

Reply:
Nice. Thanks for clarifying. Very useful feature.